Makefile

```diff
@@ -1,7 +1,7 @@
 OS ?= $(shell go env GOOS)
 ARCH ?= $(shell go env GOARCH)
 
-IMAGE_NAME := "webhook"
+IMAGE_NAME := "ghcr.io/toothstone/cert-manager-webhook-anexia"
```

Comment:
toothstone? I guess you want anexia-it here

Reply:
As with all other references to my personal Github repo/registry, I'll change this to the anexia-it locations once we get closer to actually merging. Currently these changes would break all the CI as I don't want to manually keep all the images and charts in sync.